--- auto/lib/openssl/make
+++ auto/lib/openssl/make
@@ -51,8 +51,8 @@
 $OPENSSL/.openssl/include/openssl/ssl.h:	$NGX_MAKEFILE
 	cd $OPENSSL \\
 	&& if [ -f Makefile ]; then \$(MAKE) clean; fi \\
-	&& ./config --prefix=$ngx_prefix no-shared no-threads $OPENSSL_OPT \\
-	&& \$(MAKE) \\
+	&& CFLAGS="-fPIC" CPPFLAGS="-fPIC" ./config --prefix=$ngx_prefix no-shared no-threads --with-pic $OPENSSL_OPT \\
+	&& CFLAGS="-fPIC" CPPFLAGS="-fPIC" \$(MAKE) \\
 	&& \$(MAKE) install_sw LIBDIR=lib

 END
