diff --git a/src/lib/libcrypto/x509/x509_vfy.c b/src/lib/libcrypto/x509/x509_vfy.c
index c09a2c3..d4c61d9 100644
--- a/src/lib/libcrypto/x509/x509_vfy.c
+++ b/src/lib/libcrypto/x509/x509_vfy.c
@@ -546,7 +546,15 @@ X509_verify_cert(X509_STORE_CTX *ctx)
 	/* Safety net, error returns must set ctx->error */
 	if (ok <= 0 && ctx->error == X509_V_OK)
 		ctx->error = X509_V_ERR_UNSPECIFIED;
-	return ok;
+
+	/*
+	 * Safety net, if user provided verify callback indicates sucess
+	 * make sure they have set error to X509_V_OK
+	 */
+	if (ctx->verify_cb != null_callback && ok == 1)
+		ctx->error = X509_V_OK;
+
+	return(ctx->error == X509_V_OK);
 }
 
 /* Given a STACK_OF(X509) find the issuer of cert (if any)
