--- nginx.spec
+++ nginx.spec
@@ -74,6 +74,8 @@
 Source12: COPYRIGHT
 Source13: nginx.check-reload.sh

+Patch1000: nginx-pic.patch
+
 License: 2-clause BSD-like license

 BuildRoot: %{_tmppath}/%{name}-%{main_version}-%{main_release}-root
@@ -97,6 +99,7 @@
     -e 's|%%PROVIDES%%|nginx|g' < %{SOURCE2} > nginx.init
 sed -e 's|%%DEFAULTSTART%%||g' -e 's|%%DEFAULTSTOP%%|0 1 2 3 4 5 6|g' \
     -e 's|%%PROVIDES%%|nginx-debug|g' < %{SOURCE2} > nginx-debug.init
+%patch1000 -p0

 %build
 ./configure %{BASE_CONFIGURE_ARGS} \
