--- nginx.spec
+++ nginx.spec
@@ -75,6 +75,7 @@
 Source13: nginx.check-reload.sh

 Patch1000: nginx-pic.patch
+Patch1001: nginx-cab37803ebb3.patch

 License: 2-clause BSD-like license

@@ -100,6 +101,7 @@
 sed -e 's|%%DEFAULTSTART%%||g' -e 's|%%DEFAULTSTOP%%|0 1 2 3 4 5 6|g' \
     -e 's|%%PROVIDES%%|nginx-debug|g' < %{SOURCE2} > nginx-debug.init
 %patch1000 -p0
+%patch1001 -p1

 %build
 ./configure %{BASE_CONFIGURE_ARGS} \

