--- nginx.spec
+++ nginx.spec
@@ -93,6 +93,8 @@
 Source12: nginx.copyright
 Source13: nginx.check-reload.sh

+Patch1000: nginx-pic.patch
+


 License: 2-clause BSD-like license
@@ -113,12 +115,13 @@
 %endif

 %prep
-%autosetup -p1
+%setup -q
 cp %{SOURCE2} .
 sed -e 's|%%DEFAULTSTART%%|2 3 4 5|g' -e 's|%%DEFAULTSTOP%%|0 1 6|g' \
     -e 's|%%PROVIDES%%|nginx|g' < %{SOURCE2} > nginx.init
 sed -e 's|%%DEFAULTSTART%%||g' -e 's|%%DEFAULTSTOP%%|0 1 2 3 4 5 6|g' \
     -e 's|%%PROVIDES%%|nginx-debug|g' < %{SOURCE2} > nginx-debug.init
+%patch1000 -p0

 %build
 ./configure %{BASE_CONFIGURE_ARGS} \
